src/model/content.ts

```diff
@@ -16,9 +17,9 @@ export class Content {
   public getReminders(doneOnly: boolean = true): Array<Reminder> {
     const reminders = parseReminder(this.doc);
     if (!doneOnly) {
-      return reminders;
+      return reminders.filter(reminder => reminder.status != ReminderStatus.Done && reminder.status != ReminderStatus.Cancelled);
     }
-    return reminders.filter(reminder => !reminder.done);
+    return reminders.filter(reminder => reminder.status == ReminderStatus.Done);
```

Comment:
I'm so sorry but the parameter name `doneOnly` is confusing.
The name should be `excludeDone`.

So the line should be

```
reminders.filter(reminder => reminder.status !== ReminderStatus.Done);
```

In addition, with the current implementation, `Reminders` instance keeps all reminders (including canceled, in progress) and become bigger and bigger. (may consume a lot of RAM)
I think `Reminders` should not have the canceled/in progress reminders.

I haven't had much time to review this, so please let me know if there are any misunderstandings 🙏🏼